Makefile

```diff
@@ -164,6 +166,14 @@ generate: controller-gen
 	go generate ./...
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+# Check and add (if missing) license header
+LICENSE_FILE=$(shell pwd)/LICENSE
+.PHONY: add-license 
+add-license: addlicense
+	echo "Checking/Adding license..."
```

Comment:
This `echo` should still have the `@` to prevent it printing twice.

Reply:
Oh I might have misunderstood it!